packages/build-info/src/project.ts

```diff
@@ -259,23 +269,9 @@ export class Project {
 
   async detectFrameworksInPath(path?: string): Promise<DetectedFramework[]> {
     try {
-      let detected: DetectedFramework[] = []
-      // on node we can parallelize more
-      if (this.fs.getEnvironment() === Environment.Node) {
-        detected = (await Promise.all(frameworks.map((Framework) => new Framework(this, path).detect()))).filter(
-          Boolean,
-        ) as DetectedFramework[]
-      } else {
-        // In the browser perform the detection in series to avoid having the same HTTP request in parallel.
-        // It's faster to perform one detection, do all the needed HTTP requests and reuse the results
-        // on consecutive runs.
-        for (const Framework of frameworks) {
-          const res = await new Framework(this, path).detect()
-          if (res) {
-            detected.push(res)
-          }
-        }
-      }
+      const detected = (await Promise.all(frameworks.map((Framework) => new Framework(this, path).detect()))).filter(
+        Boolean,
+      ) as DetectedFramework[]
```

Comment:
We could use `ts-reset`, but it is not meant for libraries, which I guess this qualifies for.